readme.md

```diff
@@ -20,6 +20,16 @@ This will automatically install the required dependencies.
 
 If you need to update the library, you can use `pip install --upgrade splatplost`.
 
+### Important for Use on a Linux VM (with Windows host)
+
+You need:
+A generic USB Bluetooth adapter
+VirtualBox installed on the host computer
+
+Make sure Bluetooth adapter is inserted into USB slot of host device. You should have a Linux virtual machine (Ubuntu works) running through VirtualBox, with guest additions enabled. In the VirtualBox menu, right click on the VM instance -> Settings -> USB -> Add Device to add your generic USB bluetooth adapter. On Windows host, right click on Win logo -> Device Manager -> Bluetooth, then right click on the bluetooth adapter and disable device to hand access over to the guest machine. You should now be able to enable Bluetooth in Ubuntu settings and pair to your Switch.
```

Comment:
Have you tried running the flashable image with VM? If it is applicable, it will be helpful for others to avoid the dependency installation part